.circleci/config.yml

```diff
@@ -145,6 +145,15 @@ jobs:
             -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} \
             -delete -draft v$(cat VERSION) ./artifacts
 
+  deploy-brew:
+    machine: true
+    steps:
+      - attach_workspace:
+          at: ~/workbase
+      - install-bazel-linux-rbe
+      - checkout
+      - run: DEPLOY_BREW_CHECKSUM=$(sha256sum ~/workbase/artifacts/grakn-workbase-$(cat VERSION)-mac.dmg | awk '{print $1}') bazel run //:deploy-brew -- release
```

Comment:
can we break this to multiple (simpler) lines, @vmax ?

Reply:
@haikalpribadi addressed in https://github.com/graknlabs/workbase/pull/103/commits/18955e713f867e4f059896aa65fd759f13d4ac5d